### PR TITLE
LPS-144263 Fixing "Provisioned" column acceptance criteria

### PR DESCRIPTION
**Pull request by aildsonf**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js [hunk 1]
```diff
@@ -76,6 +85,13 @@ const ModalCardSubscription = ({
 	const totalCount =
 		subscriptionsTerms?.c?.accountSubscriptionTerms?.totalCount;
 
+	const removeProvisionedColumn = () => {
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js:88]: I liked this function, but not the name, sounds like an action, but actually is returning columns

What you think about: `columnsWithoutProvisioned`

Or other name that put some emphasis in columns or a list of something

**Review by kevenleone — COMMENTED** [review]
This is not a mandatory change, but would be good

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js (hunk 1 above)
Comment (kevenleone) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js:88]: cc: @aildsonf , @jairmedeiros

Comment (aildsonf) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js]: Thanks @kevenleone for the suggestion, I strongly agree with that. It's fixed in commit ac0ed2f10fc439fff19c1fac3bbafc512380f13f

**Comment by kevenleone** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js (hunk 1 above)
Comment (kevenleone) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js]: Thank you

**Pull request closed without merging**